data/human/lusk.txt

```diff
@@ -0,0 +1,111 @@
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "BAM: Transport Lusk"
+	name "Transport Lusk to <planet>"
+	description "Transport entrepreneur Johann Lusk to a business meeting on <destination>."
+	source
+		attributes "deep"
+	destination "Ada"
+#	to offer
+#		random < 10
+	passengers 1
+	on offer
+		conversation
+			`As you enter the spaceport, your attention is immediately drawn to a somewhat disheveled man wearing a cheap black suit with a red and black checkered tie. He scrabbles on the floor on his hands and knees, desperately gathering a pile of loose papers into a briefcase. While others seem to be watching him, nobody has done anything to help.`
+			choice
+				`	(Help the man gather his belongings.)`
+					goto help
+				`	(Watch him struggle.)`
+				`	(Ignore him and move on.)`
+					decline
+
+			`	You watch the somewhat comical display of a grown man rummaging about on the floor as he finishes collecting his papers and rises shakily to his feet. He looks around, somewhat flustered, before focusing in on you and starting to walk in your direction. It seems it's too late to pretend that you weren't watching; he waves at you as he approaches.`
+				goto introduction
+			
+			label help
+			`	You join the man on the floor and help him gather the messy papers. He looks up and grunts at you appreciatively as he finishes cleaning up his mess, and you both stand up.`
+			label introduction
+			`	"Johann Lusk," the man says, by way of greeting. He smiles at you and extends a sweaty hand, evidently expecting you to shake it.`
+			choice
+				`	(Shake his hand.)`
+					goto shake
+				`	(Wave to him instead.)`
+					goto wave
+
+			label shake
+			`	You grasp the man's hand and give it a firm shake. It's clammy and a bit unpleasant, but he seems relieved that you didn't leave him hanging. You take a moment to introduce yourself as well.`
+				goto pitch
+			label wave
+			`	There's no way you're touching that greasy hand. You give him a wave instead and introduce yourself. Looking only a little crestfallen, Johann returns your wave.`
+			label pitch
+			`	"I hope I'm not being too forward here, but you have the look of a merchant captain about you," Johann says. "I hear your type is always looking for odd jobs, and it just so happens that I have one for you." He takes a moment to slick back his hair a bit before continuing. "You see, I'm in a bit of a pickle. I'm an entrepreneur of sorts and on my way to a very important investor meeting on <destination>. My previously arranged transport has abandoned me here for some reason, and I would thus be much obliged if you were to take me there." He looks at you hopefully, then adds, "there might be a very lucrative opportunity for you if you agree to take me to where I'm going. Also, I can pay you 15,000 credits."`
```

Comment:
This doesn't work for me, because he's lying about the payment that he can actually afford. I tried using `"apparent payment" 15000` at the top of the mission, but it's overridden by the actual payment (`9486`) in the `on complete` clause.